packages/eslint-plugin/src/utils/configHelpers.js

```diff
@@ -128,6 +128,21 @@ module.exports = {
         format: ['PascalCase'],
         ...(config.prefixInterface ? { prefix: ['I'] } : { custom: { regex: '^I[A-Z]', match: false } }),
       },
+      // Ignore properties that require quotes - https://typescript-eslint.io/rules/naming-convention/#ignore-properties-that-require-quotes
```

Comment:
very important addition. with the migration whole monorepo started to have issues on these kind of properties